### DISCUSSION TITLE
Minimum Layer time option?

### DISCUSSION THREAD

**Discussion by capposteve** [OrcaSlicer/OrcaSlicer#1380] [Ideas]
Something Ive noticed on a current model im working on where there are holes going through the model towards the top of the model where the layers are getting quite short. I am running Bambu PLA-CF and have had to knock the speed down to 80mm/s to have the overhangs half decent. 
However they could still be improved. What i feel like is happening is that the layers are not cooling off enough before it comes back around and prints over it thus ruining the overhangs. I think a minimum layer time would help slow just these short layers down enough to help with the overhangs and give me something better looking. However i cant seem to find a way of doing this in the current version of slicer.
![Uploading IMG_7773.PNG…]()

**Comment by julie777** [comment]
@capposteve 
Do the cooling settings in the filament profile work for you?

![image](https://github.com/SoftFever/OrcaSlicer/assets/5090005/9ceb882d-ee03-4ef3-88cd-5859bfef4f53)

I also have to change the overhang speed for the 10-25% to 80mm/s for some models.
![image](https://github.com/SoftFever/OrcaSlicer/assets/5090005/d575f32c-4cac-40ef-84ac-564b69409838)

**Reply by Soygen** [reply]
I know this is an old post, but I wanted to thank you for these settings. I've been having a disastrous time with ASA and overhangs and the above settings have my prints coming out beautifully. Can you explain how the layer time settings in the Part Cooling Fan section work? Is the fan threshold based on layer times longer or shorter than the time listed next to it?
